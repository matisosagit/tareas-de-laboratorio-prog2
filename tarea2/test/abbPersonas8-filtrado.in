crearABBPersonasVacio

filtradoPorFechaDeNacimientoABBPersonas 1/1/2025 1

crearPersona 15 Arturo Alvarez 10 03 1970
insertarPersonaABBPersonas

filtradoPorFechaDeNacimientoABBPersonas 10/03/1970 1
filtradoPorFechaDeNacimientoABBPersonas 10/03/1970 0
filtradoPorFechaDeNacimientoABBPersonas 10/03/1970 -1
filtradoPorFechaDeNacimientoABBPersonas 10/04/1970 -1

crearPersona 10 Beatriz Baez 25 07 1985
insertarPersonaABBPersonas

crearPersona 5 Carmen Cardozo 30 11 1972
insertarPersonaABBPersonas

crearPersona 1 Daniel Diaz 05 01 1988
insertarPersonaABBPersonas

crearPersona 11 Eugenia Estevez 30 11 1972
insertarPersonaABBPersonas

# 2 elementos
filtradoPorFechaDeNacimientoABBPersonas 30/11/1972 1

# 2 elementos
filtradoPorFechaDeNacimientoABBPersonas 30/11/1972 0

# 4 elementos
filtradoPorFechaDeNacimientoABBPersonas 29/11/1972 1

# 5 elementos
filtradoPorFechaDeNacimientoABBPersonas 01/01/2000 -1

liberarABBPersonas
Fin
