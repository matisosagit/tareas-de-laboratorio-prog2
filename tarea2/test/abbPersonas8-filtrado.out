1>2>3>4>5>6>Persona Arturo Alvarez
CI: 15
Fecha de Nacimiento: 10/3/1970
Perros adoptados:
7>8>Persona Arturo Alvarez
CI: 15
Fecha de Nacimiento: 10/3/1970
Perros adoptados:
9>10>11>12>13>14>15>16>17>#  2 elementos.
18>Persona Daniel Diaz
CI: 1
Fecha de Nacimiento: 5/1/1988
Perros adoptados:
Persona Beatriz Baez
CI: 10
Fecha de Nacimiento: 25/7/1985
Perros adoptados:
19>#  2 elementos.
20>Persona Carmen Cardozo
CI: 5
Fecha de Nacimiento: 30/11/1972
Perros adoptados:
Persona Eugenia Estevez
CI: 11
Fecha de Nacimiento: 30/11/1972
Perros adoptados:
21>#  4 elementos.
22>Persona Daniel Diaz
CI: 1
Fecha de Nacimiento: 5/1/1988
Perros adoptados:
Persona Carmen Cardozo
CI: 5
Fecha de Nacimiento: 30/11/1972
Perros adoptados:
Persona Beatriz Baez
CI: 10
Fecha de Nacimiento: 25/7/1985
Perros adoptados:
Persona Eugenia Estevez
CI: 11
Fecha de Nacimiento: 30/11/1972
Perros adoptados:
23>#  5 elementos.
24>Persona Daniel Diaz
CI: 1
Fecha de Nacimiento: 5/1/1988
Perros adoptados:
Persona Carmen Cardozo
CI: 5
Fecha de Nacimiento: 30/11/1972
Perros adoptados:
Persona Beatriz Baez
CI: 10
Fecha de Nacimiento: 25/7/1985
Perros adoptados:
Persona Eugenia Estevez
CI: 11
Fecha de Nacimiento: 30/11/1972
Perros adoptados:
Persona Arturo Alvarez
CI: 15
Fecha de Nacimiento: 10/3/1970
Perros adoptados:
25>26>Fin.
